templates/apigear/olink/tests/CMakeLists.txt

```diff
@@ -14,7 +14,6 @@ find_package(trompeloeil REQUIRED)
 set(TEST_POCO_OLINK_SOURCES
     test_main.cpp
     olink_connection.test.cpp
-    olinkhost.test.cpp
```

Comment:
why are you removing this file from the tests target?
And in case it is not needed anymore, it should also be deleted from the repository